generals/rewards/common.py

```diff
@@ -0,0 +1,33 @@
+"""
+This module is for reuseable functions that aid in creating reward-functions.
+For example, computing the number of cities owned by an agent based on an observation.
+"""
+
+from generals.core.action import Action, compute_valid_move_mask
+from generals.core.observation import Observation
+
+
+def compute_num_cities_owned(observation: Observation) -> int:
+    owned_cities_mask = observation.cities & observation.owned_cells
+    num_cities_owned = owned_cities_mask.sum()
+    return num_cities_owned
+
+
+def compute_num_generals_owned(observation: Observation) -> int:
```

Comment:
I now recall that you once asked whether this repo supports FFA and I forgot to reply. Currently it does not, and I don't plan to do it personally, unless some external force makes me :D So in this repository this is just 1 general = still playing, 2 generals = win, 0 generals = lose. I think you already know it, just making a comment :)

Reply:
Mhm, thanks for the heads up! If I can ever get my 1v1 bot working, I may try sorting out an FFA-capable one.